src/commands/lightning/preview/site.ts

```diff
@@ -21,13 +21,11 @@ export type LightningPreviewSiteResult = {
 
 export default class LightningPreviewSite extends SfCommand<LightningPreviewSiteResult> {
   public static readonly summary = messages.getMessage('summary');
-  public static readonly description = messages.getMessage('description');
```

Comment:
@nrkruk I wanted to delete the `description` for the `--name` flag, but I had to delete these two lines of your TypeScript to do so. Just flagging this to make sure those deletions don't affect anything.